torch/_prims/__init__.py

```diff
@@ -329,6 +329,19 @@ def _backend_select_impl(*args, **kwargs):
     _prim = _prim_packet.default
     if tags:
         _prim._tags = tags
+    elif aten_packet := getattr(torch.ops.aten, name, None):
+        overload_tags = [
+            getattr(aten_packet, overload).tags for overload in aten_packet.overloads()
+        ]
+        tags_intersection = set(overload_tags[0])
```

Comment:
Can't you just use your new OrderedSet instead

Reply:
not landed yet.. can do in follow ups